### ISSUE TITLE
Progress bar doesn't work correctly

### ISSUE THREAD

**Issue by akshat1** [akshat1/video-hoarder#12]
The download progress bar always shows 50%.

(That's because I haven't actually implemented the plumbing to make it work, but creating this bug here as a placeholder).

**Comment by akshat1** [comment]
The fix works when there's just one video in the queue. However with multiple videos the progress bar updates sporadically and doesn't go all the way to 100% when the download completes.

![image](https://user-images.githubusercontent.com/1529776/67139122-36376580-f201-11e9-90ff-cb5a1ced46c5.png)

**Comment by akshat1** [comment]
I fixed the latest progress bar issue only to realise I don't have a good way to make progress bars work with split downloads (where download separate video/audio files). This is because we get our progress values by regexing the ytdl progress output; however ytdl messages go from 0 - 100% for _each_ part of the download. This causes out download to go from 0 - 100, then back to 0, then 100.

There was another issue where we fallback to a default empty progress information (instead of the last known progress status). That one is easily fixable but not worth it until the first problem is solved. I have removed the progress bars (#20), and I'm closing this ticket for now.
